bin/poweroff.digital-loggers

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+curl --data ${power_port}=OFF -o /dev/null --silent \
+	http://admin:1234@${power_ip}/outlet
```

Comment:
I guess those values must be the HW default. The security nazi in me wants to make it difficult to use those values to force people to change the default password:-) However, I guess it's reasonable for the script to implement the defaults as you described.

Could the variables be more obviously associated with this particular script (e.g. digital_loggers_user) to avoid any possible naming conflicts if some board ends up pulling in a variety of different scripts that each talks to different equipment, thus potentially requiring multiple user/pass combos?